platformio_tubes.ini

```diff
@@ -14,43 +29,100 @@ build_flags = -O2
   -D WLED_DISABLE_HUESYNC
   -D WLED_DISABLE_WEBSOCKETS
   -D WLED_DISABLE_ADALIGHT
+  -D WLED_DISABLE_ESPNOW
   -D IRTYPE=0
 lib_ignore =
   ESPAsyncTCP
   ESPAsyncUDP
   IRremoteESP8266
 lib_deps = 
+  fastled/FastLED @ ^3.7.0
+  makuna/NeoPixelBus @ ^2.7.8
+  https://github.com/Aircoookie/ESPAsyncWebServer.git @ 2.2.1
   gmag11/QuickEspNow @ ^0.6.2
   gmag11/QuickDebug @ ^0.7.0
 
 [tubes]
 extends = tubes_no_mic
-build_flags = ${tubes_no_mic.build_flags}
-  ${esp32.AR_build_flags}  
+build_flags =
+  ${tubes_no_mic.build_flags}
+  ${esp32.AR_build_flags}
 lib_deps = ${tubes_no_mic.lib_deps}
-  ${esp32.AR_lib_deps}
+  IRremoteESP8266 @ ^2.8.6
+  https://github.com/kosme/arduinoFFT @ 2.0.2 ; ${esp32.AR_LIB_deps}
+
 
 [env:esp32_quinled_dig2go]
+; basis quinled dig2go without any tubes support
 extends = env:esp32_quinled_diguno
-lib_ignore = 
-  ${env:esp32_quinled_diguno.lib_ignore}
+build_unflags = ${env:esp32_quinled_diguno.build_unflags}
+  -D WLED_DISABLE_INFRARED
+  -D IRTYPE
+lib_ignore = ${env:esp32_quinled_diguno.lib_ignore}
 lib_deps = ${env:esp32_quinled_diguno.lib_deps}
   IRremoteESP8266 @ 2.8.6
 
 [env:esp32_quinled_dig2go_tubes]
 extends = env:esp32_quinled_dig2go
+platform = ${tubes.platform}
+platform_packages = ${tubes.platform_packages}
 build_unflags = 
-  -D WLED_DISABLE_INFRARED
-  -D IRTYPE=0
+  ${tubes.build_unflags}
+  ${env:esp32_quinled_dig2go}
 build_flags = 
   ${tubes.build_flags}
   ${env:esp32_quinled_dig2go.build_flags} 
+  -D FASTLED_ESP32_SPI_BUS=HSPI
   -D NUM_STRIPS=1 -D DEFAULT_LED_COUNT=150
 lib_ignore = 
   ESPAsyncTCP
   ESPAsyncUDP
+  ${env:esp32_quinled_dig2go.lib_ignore}
 lib_deps = 
   ${tubes.lib_deps}
-  ${env:esp32_quinled_dig2go.lib_deps} 
 
 
+
+# ------------------------------------------------------------------------------
+# ESP32 S3 Matrix M1
+# ------------------------------------------------------------------------------
+[env:esp32-s3-matrix-m1]
+; builds using the default WLED settings
+extends = env:esp32s3dev_8MB_PSRAM_opi
+board_build.arduino.memory_type = qio_qspi ;; use with PSRAM: 2MB or  4MB
+board_upload.flash_size = 4MB
+board_build.partitions = tools/WLED_ESP32_4MB_1MB_FS.csv
+build_unflags = ${env:esp32s3dev_8MB_PSRAM_opi.build_unflags} 
+  -D ARDUINO_USB_CDC_ON_BOOT=1
+  -D ARDUINO_USB_MSC_ON_BOOT=0 
+  -D ARDUINO_DFU_ON_BOOT=0
+build_flags = ${common.build_flags} ${esp32s3.build_flags} -D WLED_RELEASE_NAME=esp32-s3-matrix-m1
+  -D WLED_WATCHDOG_TIMEOUT=0
+  -D ARDUINO_USB_CDC_ON_BOOT=1
+  -D ARDUINO_USB_MODE=1
+  -D BOARD_HAS_PSRAM ; tells WLED that PSRAM shall be used
```

Comment:
Hint: 

You probably need '-D WLED_USE_PSRAM' too. Otherwise the WLED core will not use any PSRAM.

https://github.com/SteveEisner/WLEDtubes/blob/ee318dc9b886a57c4f4762cb2a193832798d8aa1/wled00/wled.h#L157

Reply:
thanks for the heads up!  You're right, we should add this.

Craig - softhack007 is one of the maintainers of the advanced WLED branch "Moon Modules", you might want to check https://mm.kno.wled.ge/advanced/buildflags/ for additional flags that are useful for this device

Reply:
I'll merge this and then push that additional change.